benchopt/cli/main.py

```diff
@@ -93,7 +97,14 @@
 def run(benchmark, solver_names, forced_solvers, dataset_names,
         objective_filters, max_runs, n_repetitions, timeout,
         plot=True, html=True, pdb=False, do_profile=False,
-        env_name='False'):
+        env_name='False', old_objective_filters=None):
+    if len(old_objective_filters):
+        print("should warn but nothing is printed")
+        warnings.warn(
```

Comment:
@agramfort do you know why the warning is captured ? The print is executed if you use `-p` but the warning is not displayed